src/stub_test.py

```diff
@@ -0,0 +1,23 @@
+#  Copyright 2020 Waterball (johnny850807@gmail.com)
+#
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+from unittest import TestCase
+
+from src import stub
+
```

Comment:
Nothing to do with this file, just to mention that `pytest` might be simpler in some cases. It can save some time without writing entry points for test cases.

Reply:
I agree ... :) Let's go for `pytest`